[bug] ルールを更新できない

**Describe the bug**
`-u`を実行したら、以下のエラーで失敗します：
```
./hayabusa-1.2.0-mac-intel -u -q
Attempting to git clone the hayabusa-rules repository into the rules folder.
[ERROR] Failed to git clone into the rules folder. Please rename your rules folder name. 'rules' exists and is not an empty directory; class=Invalid (3); code=Exists (-4)
[ERROR] Failed to update rules. Error { code: -1, klass: 0, message: "" }
```

**Step to Reproduce**
Steps to reproduce the behavior:
v.1.2.0のリリース.zipをダウンロードして、そのまま更新しようとしています。

**Environment (please complete the following information):**
 - OS: macOS
 - hayabusa version 1.2.0

--- a/src/main.rs
+++ b/src/main.rs
@@ -667,8 +667,6 @@ impl App {
             // case of no exist hayabusa-rules repository in rules.
             // execute update because submodule information exists if hayabusa repository exists submodule information.
 
-            self._repo_main_reset_hard(hayabusa_rule_repo.as_ref().unwrap())?;
-
             prev_modified_time = fs::metadata("rules").unwrap().modified().unwrap();
             let rules_path = Path::new("rules");
             if !rules_path.exists() {

